jmxtrans-core/src/main/java/com/googlecode/jmxtrans/JmxTransformer.java

```diff
@@ -159,7 +172,7 @@ public static void main(String[] args) throws Exception {
 	 */
 	private void doMain() throws Exception {
 		// Start the process
-		this.start();
+		start();
```

Comment:
There are many unneeded changes, mostly code style, in this PR. Removing `this.` is not needed and adds unnecessary noise to this PR. Maybe move this to another PR.